### ISSUE TITLE
addButton() do not work after update

### ISSUE THREAD

**Issue by FenziRocks** [trigger-corp/trigger.io-tabs#18]
**Device information:**

Hardware: Iphone 11
Operating System and Version: iOS 13.1
Forge Platform version: v2.8
App information:
```
{
  "config_version": "4",
  "version": "4.1.300",
  "platform_version": "v2.8.1",
 
 },
    "tabs": {
      "version": "3.1",
      "config": {},
}
```

**To Reproduce**

- open a native tab
- the added button appears
- hit the added button
- nothing happens

**Expected behavior** (with the older versions platform 2.7 and tabs version 2.32.25)
The functionality should be executed. We are using nativeTab.executeJS().

**Actual behaviour**
Nothing happens.
No error is thrown even in catalyst we cannot see any error.

**Comment by antoinevg** [comment]
Thank you for reporting this!

I've pushed a fix to `v3.2` of the module.

**Comment by FenziRocks** [comment]
Super, we have test it and it looks good so far.
Thank you very much for the quick response and the solution.
